torch/_inductor/codecache.py

```diff
@@ -379,6 +386,240 @@ def write_atomic(path: str, content: Union[str, bytes]) -> None:
     tmp_path.rename(path)
 
 
+@dataclasses.dataclass
+class TensorMetadata:
+    """
+    The Tensor metadata relevant when hashing FxGraph cache keys.
```

Comment:
Please let me know if there are other tensor fields that are relevant to hashing